qa/rpc-tests/wallet_large.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python2
+# Copyright (c) 2017 The Zcash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#
+# Create a large wallet
+#
+# To use:
+# - Copy to qa/rpc-tests/wallet_large.py
+# - Add wallet_large.py to RPC tests list
+# - ./qa/pull-tester/rpc-tests.sh wallet_large --noshutdown
+# - Archive the resulting /tmp/test###### directory
+#
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+    assert_equal,
+    connect_nodes_bi,
+    initialize_chain_clean,
+    start_nodes,
+)
+
+from decimal import Decimal
+
+
+class LargeWalletTest(BitcoinTestFramework):
```

Comment:
This test doesn't seem to be related to the subject of the PR.

Reply:
force-pushed fix